Makefile

```diff
@@ -167,6 +167,36 @@ sumo_tarball_latest: checkout_leapdeps_develop pull_leapdeps setup_without_names
 pyinst:
 	pyinstaller -y pkg/pyinst/bitmask.spec
 
+pyinst-hacks:
+	mkdir -p dist/bitmask/cryptography/hazmat/primitives/src/
+	cp ~/src/cryptography-0.9.3/src/cryptography/hazmat/primitives/src/* dist/bitmask/cryptography/hazmat/primitives/src/
```

Comment:
uh. it turns out I had an old cryptography version in my venv, that's why I needed the hacks.
newer versions get this shit included properly via pyinstaller hooks.
removing, thanks for catching this.